ovos_number_parser/numbers_pt.py

```
@@ -446,10 +447,12 @@ def numbers_to_digits_pt(utterance: str) -> str:
         "zero": "0"
     }
-    words = tokenize(utterance)
-    for idx, word in enumerate(words):
-        if word in number_replacements:
-            words[idx] = number_replacements[word]
+    words: List[Token] = tokenize(utterance)
+    for idx, tok in enumerate(words):
+        if tok.word in number_replacements:
+            words[idx] = number_replacements[tok.word]
+        else:
+            words[idx] = tok.word
```

Ruff (0.8.2)

Use `words[idx] = number_replacements.get(tok.word, tok.word)` instead of an `if` block

Replace with `words[idx] = number_replacements.get(tok.word, tok.word)`

(SIM401)